src/Stack/Options.hs

```diff
@@ -861,7 +861,7 @@ ghcVariantParser hide =
         readGHCVariant
         (long "ghc-variant" <> metavar "VARIANT" <>
          help
-             "Specialized GHC variant, e.g. integersimple (implies --no-system-ghc)" <>
+             "Specialized GHC variant, e.g. integersimple" <>
```

Comment:
That's still implied, no? Or is it now legal to use `--system-ghc=... --ghc-variant`?